components/gallery/hero/hero.css

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
look at how commerce and lune templates have introduced a partial/ folder for template specific components.